app/models/host/managed.rb

```diff
@@ -1055,5 +1055,7 @@ def extract_params_from_object_ancestors(object)
   def send_built_notification
     recipients = owner ? owner.recipients_for(:host_built) : []
     MailNotification[:host_built].deliver(self, :users => recipients) if recipients.present?
+  rescue SocketError, Net::SMTPAuthenticationError, Net::SMTPServerBusy, Net::SMTPSyntaxError, Net::SMTPFatalError, Net::SMTPUnknownError => e
+    logger.debug "Host has been created. Failed to send email with #{e}"
```

Comment:
1. Should be at least warning level, nobody will know it's a problem if at debug.
2. Use the exception logger.